cataloging/src/components/inspector/field.vue

```diff
@@ -504,7 +508,96 @@ export default {
     },
     isAnyType() {
       return this.entityType === ANY_TYPE;
-    }
+    },
+    hasFind() {
+      if (!this.itemData) return false
+
+      let find = null
+
+      if(this.archType === 'Instance') {
+        find = this.itemData?.instanceOf?._categoryByCollection?.find;
+      }
```

Comment:
Instead of getting `_categoryByCollection` from the record, you can check if  `this.fieldKey === _categoryByCollection`. `Find`, `identify` and` @none` can be obtained from `this.fieldValue`. You can check `this.parentPath` to tell if this is category in the local work on the instance or on the instance level.